DataSetExplorer/DataSetSerializer/IDataSetExporter.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace DataSetExplorer
+{
+    public interface IDataSetExporter
```

Comment:
Maybe we should include the keyword Service for all of these interfaces (including the earlier Creator interface I made).

I can see how this interface can be used for a low-level technical implementation (e.g., ExcelExporter), but our aim is to have a full use case implementation.

For domain services we don't want to use the keyword Service, so the IFullDataSetBuilder should remain the same. This will be helpful to differentiate domain and application services.